freshmaker/lightblue.py

```diff
@@ -996,21 +996,23 @@ def find_images_with_included_rpms(
             name = koji.parse_NVR(rpm_nvr)["name"]
             rpm_name_to_nvrs.setdefault(name, []).append(rpm_nvr)
 
-        image_request = {
-            "objectType": "containerImage",
-            "query": {},   # set by _set_container_image_filters()
-            "projection": self._get_default_projection(
-                rpm_names=rpm_name_to_nvrs.keys(),
-                include_rpm_manifest=include_rpm_manifest)
-        }
+        rpm_names = [kobo.rpmlib.parse_nvr(nvr)["name"] for nvr in rpm_nvrs]
+        images = self.pyxis.find_images_by_installed_rpms(rpm_name_to_nvrs.keys(), rpm_names, content_sets, repositories, published)
```

Comment:
Check the definition:
```
  def find_images_by_installed_rpms(                                                                                                                                                               
      self, rpm_names, content_sets=None, repositories=None, published=None, tags=None
  ):
```

Reply:
https://github.com/redhat-exd-rebuilds/freshmaker/pull/203/files#diff-9ccbae0ffc4e5b4c63410f2592bc4a2088f97c3b45ec55ea5a6f793258038248R1000
auto_rebuild_tags added